src/react-components/ui-root.js

```diff
@@ -1232,7 +1299,14 @@ class UIRoot extends Component {
             </div>
 
             {this.state.showPresenceList && (
-              <PresenceList presences={this.props.presences} sessionId={this.props.sessionId} />
+              <PresenceList
+                presences={this.props.presences}
+                sessionId={this.props.sessionId}
+                signedIn={this.state.signedIn}
+                email={this.props.store.state.credentials.email}
```

Comment:
could `credentials` be undefined here for existing users?

Reply:
Good question. I'll check, but I think the change I made in `store.js` covers that.